mongoOperator/helpers/listeners/mongo/CommandLogger.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2018 Ultimaker
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+import logging
+
+from pymongo.monitoring import CommandStartedEvent, CommandListener, CommandSucceededEvent, CommandFailedEvent
+
+
+class CommandLogger(CommandListener):
```

Comment:
Where's this used?

Reply:
Purely for having debug logging when running the operator.